components/courses/spec/models/courses/progress_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe Courses::Progress, type: :model do
+  let!(:season)   { create(:season, title: 'Test Season') }
+  let!(:user)     { create(:user) }
+  let!(:venue)    { create(:venue) }
+  let!(:mentor)   { ::Courses::Mentor.create(user: user, season: season) }
+  let!(:lecture)  { create(:lecture, mentor: mentor, venue: venue, season: season) }
+  let!(:student)  { ::Courses::Student.create(season: season, user: user, status: :attending,
+                                              personal_info: 'lorem', motivation_info: 'ipsum') }
+  let!(:progress) { ::Courses::Progress.create(student: student, lecture: lecture, mentor: mentor, homework_mark: 2) }
+
+  describe 'validations' do
+    it 'should pass, due to correct homework mark input' do
```

Comment:
`shoulda-matchers` gem has very useful matchers for the inclusion testing.
Check it out - https://github.com/thoughtbot/shoulda-matchers/blob/master/lib/shoulda/matchers/active_model/validate_inclusion_of_matcher.rb#L22